src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -314,6 +314,13 @@
     private YesNo showSchedule3ActivitiesPage;
     private YesNo doesRegulation35Apply;
     private YesNo showFinalDecisionNoticeSummaryOfOutcomePage;
+    private YesNo doesSchedule8Paragraph4Apply;
+    private YesNo showSchedule8Paragraph4Page;
+    private YesNo showSchedule7ActivitiesPage;
+    private YesNo doesSchedule9Paragraph4Apply;
+
+    @JsonUnwrapped
+    private SscsUcCaseData sscsUcCaseData;
```

Comment:
Nice to have. This could annotate be with  `@Getter(AccessLevel.NONE)` so that lombok doesn't build the getter.

Reply:
Good idea! - I hadn't used that annotation before - thanks!